src/pages/FirstUsePage.js

```diff
@@ -162,7 +163,23 @@ export class FirstUsePage extends React.Component {
             />
           </View>
         </View>
+        <View style={localStyles.demoSiteRequestButtonContainer}>
+          <View style={globalStyles.horizontalContainer}>
+            <Button
+              style={[globalStyles.authFormButton, { flex: 1 }]}
+              textStyle={globalStyles.authFormButtonText}
+              text="Request a Demo Store"
+              onPress={() => this.setState({ isDemoUserModalOpen: true })}
```

Comment:
having the method defined like this ( () => {} ) within the render method will create a new method each time the component is rendered, which is a small performance hit.

Could you change to create a parameterless method in the class, and call it directly:

handleDemoModalOpen = () => this.setState((prev, props) => ({ isDemoUserModalOpen: true }));

onPress={this.handleDemoModalOpen}

Reply:
Please check https://github.com/sussol/mobile/pull/724/commits/4eebfeb8ed6e6df6d5297d18d5f7e3b54266475a

Reply:
@mark-prins FYI there's a babel transform called reflective-bind that allows you to keep this nice syntax without the recreation every render - check it out https://flexport.engineering/ending-the-debate-on-inline-functions-in-react-8c03fabd144